File: app/src/main/java/com/inmo/component/MainActivity.java
package com.inmo.component;

import android.os.Bundle;

import androidx.annotation.Nullable;

import com.inmoair.ui.dialog.AnimConfirmCancelDialog;
import com.inmoair.ui.toast.AirToast;

/**
 * @author Administrator
 */
public class MainActivity extends BaseActivity {

    private AnimConfirmCancelDialog mConfirmCancelDialog;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
//        AirToast.showToast("蓝牙已连接");
//        AirToast.showIconToast(this, "蓝牙已连接", R.mipmap.ic_launcher);
        showAnimDialog();
    }

    private void showAnimDialog() {
        if (mConfirmCancelDialog == null) {
            mConfirmCancelDialog = new AnimConfirmCancelDialog(MainActivity.this, true);
        }

        if (mConfirmCancelDialog.isShowing()) {
            return;
        }
        mConfirmCancelDialog.setBtnText("取消", "确认");
        mConfirmCancelDialog.show();
        mConfirmCancelDialog.setTitle("确定把当前选中的9张美美照片同步到手机吗确定把当前选中的9张美美照片同步到手机?");
        mConfirmCancelDialog.setOnScrollListener(new AnimConfirmCancelDialog.OnScrollListener() {

            @Override
            public void ensure() {
                mConfirmCancelDialog.dismiss();
            }

            @Override
            public void cancel() {
                mConfirmCancelDialog.dismiss();
            }
        });
    }
}

File: app/src/main/java/com/inmo/component/MainActivity.kt
package com.inmo.component

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        addConfirmCancelDialogBelowProgress()
        btn_ui.setOnClickListener {
            val intent = Intent(this, UiActivity::class.java)
            startActivity(intent)
        }
        btn_net.setOnClickListener {
            val intent = Intent(this, NetworkActivity::class.java)
            startActivity(intent)
        }
    }

    private fun addConfirmCancelDialogBelowProgress() {
        // ConfirmCancelDialog
//        ConfirmCancelDialog commonDialog = new ConfirmCancelDialog.Builder(this)
//                .setListener(new ConfirmCancelDialog.Builder.DialogListener() {
//                    @Override
//                    public void sure() {
//                        // TODO: 2021/11/2
//                    }
//
//                    @Override
//                    public void cancel() {
//                        // TODO: 2021/11/2
//                    }
//                })
//                .setCloseIconVisible(false)
//                .setTitleVisible(false)
//                .setCancelBtnVisible(false)
//                .setTitle("更新通知")
//                .setContent("当前版本暂不可用，请下载最新版本，以便享受优质内容。")
//                .build();
//        commonDialog.show();
//        val loadingDialog = LoadingDialog(this@MainActivity)
//        loadingDialog.show("", false, false)
    }
}